app/components/spin-button.js

```diff
@@ -90,16 +91,19 @@ export default Ember.Component.extend({
       this.setDisabled();
 
       if (this.get('startDelay') > 4) {
-        Ember.run.later(this, this.createSpinner, element, this.get('startDelay'));
+       this.set('_spinnerTimer', Ember.run.later(this, this.createSpinner, element, this.get('startDelay')));
       }else{
         this.createSpinner(element);
       }
     }else{
+      Ember.run.cancel(this.get('_spinnerTimer'));
+      this.set('_spinnerTimer', null);
       this.setEnabled();
     }
   }.observes('inFlight'),
 
   createSpinner: function(element) {
+    this.set('inFlightStyle', true);
```

Comment:
Yeah it's a bit confusing, the term is taken from Ember Data, where `inFlight` indicates that a model is transitioning. I used the term here to indicate that we should display the spinner and disable the button. It's actually a public API, and is used publicly in a few places, even though I planned to deprecate it. 

When set to true, it adds the 'in-flight' class using bindings, as ember interpretes true to mean "use the variable name, not the literal value `true`".

You've broken the public API because there's now two different properties which are meant to do the same thing.